MapGameController.java

```diff
@@ -354,6 +356,10 @@ public void afterMove(){
             initCount++;
             floor.setText("B"+initCount);
             init();
+        }else if(chara.getPosX() == enemy.getPosX() && chara.getPosY() == enemy.getPosY()){
+            se.play();
+            Wiz.setImage(new Image (getClass().getResourceAsStream("pic/cat.jpg")));
+            timeline.stop();
```

Comment:
Why does it stop?

Reply:
I thought  to make it alternative just like invincible time.

Reply:
You mean that the enemy stops for a while once it touches the player?

Reply:
In the current implementation, the enemy keep stopped until the player proceeds to the next floor.

Reply:
Yes. 
But, I didn' t have idea after stopping, so I  did'nt anything.

Reply:
Probably you want to make new boolean field to keep the state.